ReactiveCocoa/ControlSubject.swift

```diff
@@ -0,0 +1,44 @@
+import ReactiveSwift
+import enum Result.NoError
+
+/// Represents a subject of an interactive control.
+///
+/// Generally speaking, `ControlSubject` emits changes only for the user
+/// interactions in which the subject specializes.
+public final class ControlSubject<Value> {
```

Comment:
@mdiep The problem here is that the assumed behavior of `Property` does not match how `Control` behaves.

1. For every value you set, a `next` event is emitted by the property. For `UIControl`, this never happens for any values being set manually.
2. Because of (1), the latest value emitted by `producer` is NOT guaranteed to be `value` in the context of `UIControl`.